rpc_util.go

```diff
@@ -55,13 +66,28 @@ type gzipCompressor struct {
 
 // NewGZIPCompressor creates a Compressor based on GZIP.
 func NewGZIPCompressor() Compressor {
+	c, _ := NewGZIPCompressorWithLevel(BestCompression)
```

Comment:
We should use `gzip.DefaultCompression` here to avoid affecting users already expecting it.